pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -238,7 +245,7 @@ func (b *volumeBinder) FindPodVolumes(pod *v1.Pod, node *v1.Node) (unboundVolume
 			if selectedNode, ok := claim.Annotations[pvutil.AnnSelectedNode]; ok {
 				if selectedNode != node.Name {
 					// Fast path, skip unmatched node
-					return false, boundVolumesSatisfied, nil
+					return []string{ErrReasonBindConflict}, nil
```

Comment:
I need to point out one minor simplification: previously, both booleans might have been `false` here. Now only `ErrReasonBindConflict` is returned as reason for the scheduling failure.

None of the unit tests covered that particular combination. I think its enough to return one reason here.